app/models/metric/processing.rb

```diff
@@ -161,7 +164,7 @@ def self.interval_name_to_interval(name)
     when "realtime" then 20
     when "hourly" then   1.hour.to_i
     when "daily" then    1.day.to_i
-    else             raise "unknown interval name: [#{name}]"
+    else  raise _("unknown interval name: [%{name}]") % {:name => name}
```

Comment:
There's one extra space between `else` and 'raise` here, this should be fixed. Otherwise the PR looks good.